src/router/routes.ts

```diff
@@ -9,7 +6,9 @@ import SignoutView from '@/views/auth/Signout.vue'
 export const routes = [
   {
     path: '/',
-    redirect: '/define-company'
+    meta: {
+      requiresAuth: true
+    }
```

Comment:
Good change but this raises an issue elsewhere.

Normally, the auth process (signin page) would redirect back to the starting route (ie, /), but this route doesn't do anything. So I propose that / continue to redirect to define-company, and that the router code / signin page redirect back to the starting route; this can be done by passing the starting route as a param / prop to the signin component.

Reply:
Okie doke, i will try this out!

Reply:
Can you debug this and figure out what's happening?

Use cases: user should be able to:
- browse to http://our_app/, login if needed, and return to / (which logically should redirect to /define-company
- browse to http://our_app/review-confirm, login if needed, and return to /review-confirm

Reply:
You bet, looking at this now!

Reply:
Cam mentioned this is working atm.